src/webflask.ts

```diff
@@ -195,18 +246,25 @@ export class WebFlask {
       method = init.method;
     }
 
-    return dedent`
-      with app.server.app_context():
-        with app.server.test_client() as client:
-          response = client.open('${info}',
-            data=${data},
-            content_type=${content_type},
-            method="${method}",
-          )
-      if response.status_code == 424:
-        __import__(response.get_data(as_text=True))
-      response
+    if (this.python_lazy_loading) {
```

Comment:
I think there should be a comment here to state that this is initialised after xiplot just in time before the dash app is bootstrapped, so that initial loads must succeed but user-triggered ones in the frontend can be lazy. Given this context, I'm unsure if a global initialisation flag is the best idea here ...

Reply:
I agree that this design is very flexible - however we are using it for a specific purpose, which is catching all lazy loads caused by user interaction. I think there should be a comment either here or at the init call saying this